design/paging.md

```diff
@@ -0,0 +1,119 @@
+# Paging
+
+## Document purpose
+
+Describe options for paging through collections data (although this discussion is generally
+applicable to any data stored in a database that is too large to return to the user at once)
+and their pros and cons.
+
+Paging is often an afterthought or considered a non-issue, but this is not the case, and there
+is no ideal solution that the author is aware of. Database schemas and APIs may need modification
+to page effectively and efficiently if not designed with that in mind.
+
+## Nomenclature
+
+* `documents` can be read as documents in in a document based database, like MongoDB or ArangoDB,
+  or rows in a RDBMS, like MySQL or Postgres.
+
+## Options
+
+### Skip and limit / offset and limit
+
+This is the standard paging solution of providing a count of documents to skip and a count
+of documents to return.
+
+#### Pros
+
+* Can page to any location in the data
+* Easily understood by users
```

Comment:
I'm not sure they pay too much attention to anything besides the page number. Is that what you're getting at here?

Reply:
It's just that they're familiar with it and aren't likely to be confused when presented with the UI